easybuild/easyconfigs/p/PortAudio/PortAudio-19.7.0-foss-2022a.eb

```diff
@@ -0,0 +1,52 @@
+easyblock = 'ConfigureMake'
```

Comment:
I can see why you've done it this way, to encompass the Python and non-Python parts.
Might be worth a comment to explain this approach, because otherwise it looks quite strange.

Reply:
Actually I don't really understand why there is also `pyaudio` in this easyconfig. I copied this from another repository of easyconfigs: https://github.com/ComputeCanada/easybuild-easyconfigs/blob/computecanada-main/easybuild/easyconfigs/p/PortAudio/PortAudio-190700_20210406-GCCcore-9.3.0.eb
The `pyaudio` package provides Python bindings for `PortAudio` but it is not needed by `EasyMocap` so at least `pyaudio` should have it's own easyconfig in my opinion. What do you thing?

Reply:
There's no harm in including the Python bindings for something along with the installation itself.
If including `pyaudio` in here doesn't cause trouble, then it's a sane approach I think.

I don't think there's a real need here to explain why `ConfigureMake` is used as easyblock: that is responsible for installing `include/*` and `lib/*`, while `pyaudio` takes care of `lib/python*/site-packages` (that should be listed in `sanity_check_paths` too, perhaps)